sdk/src/errors.rs

```diff
@@ -225,9 +225,9 @@ pub(crate) fn status_to_error(status: tonic::Status) -> MomentoError {
                                 inner_error: Some(status.clone().into()),
                                 details: Some(status.into())
                             },
-                            unknown_err => MomentoError {
-                                message: format!("Unknown error has occurred, unsupported error type for NotFound: {:?}", unknown_err),
-                                error_code: MomentoErrorCode::UnknownError,
+                            _ => MomentoError {
```

Comment:
This was necessary because the server has changed the err metadata since last time we committed to this SDK. Several tests were failing that were expecting CacheNotFoundError for this case; this fixes them.